src/card/dto/response-card.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { CreateCardDto } from './create-card.dto';
+import { PartialType } from '@nestjs/mapped-types';
+import {
+    IsNotEmpty,
+    IsString,
+} from 'class-validator';
+
+export class ResponseCardDto extends PartialType(CreateCardDto) {
+    @IsString()
+    @IsNotEmpty()
+    id: string;
+
+    constructor(id: string, name: string) {
+        super()
```

Comment:
It would make sence to call a full parent constructor here, something like `super(id, name)` and drop the rest code

Reply:
test it

Reply:
Looks like it works